### PR TITLE
fix(wallet-mobile): Use absolute slot number when creating a TX

### PR DESCRIPTION
**Pull request by michalszorad**
## Description / Change(s) / Related issue(s)

Use absolute slot number when creating a TX

##### Ticket

[YOMO-1932](https://emurgo.atlassian.net/browse/YOMO-1932)

[YOMO-1932]: https://emurgo.atlassian.net/browse/YOMO-1932?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/types/src/network/manager.ts [hunk 1]
```diff
@@ -53,11 +53,13 @@ export type NetworkEpochInfo = {
   start: Date
   end: Date
   era: NetworkEraConfig
+  eras: ReadonlyArray<NetworkEraConfig>
```
Comment (michalszorad) [inline packages/types/src/network/manager.ts:56]: Maybe a better name - `previousEras`?

Comment (jorbuedo) [reply packages/types/src/network/manager.ts:56]: Does it include the current one?

Comment (stackchain) [reply packages/types/src/network/manager.ts:56]: I agree

Comment (michalszorad) [reply packages/types/src/network/manager.ts:56]: Yes

File: apps/wallet-mobile/src/features/WalletManager/network-manager/helpers/epoch-progress.ts [hunk 2]
```diff
@@ -38,6 +50,7 @@ export function epochProgress(epochInfo: Network.EpochInfo) {
       {
         progress: progress,
         currentSlot: currentSlot,
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/WalletManager/network-manager/helpers/epoch-progress.ts]: my stuff 🤦🏻 
```suggestion
        progress,
        currentSlot,
```

File: apps/wallet-mobile/src/features/WalletManager/network-manager/helpers/epoch-progress.ts [hunk 3]
```diff
@@ -38,6 +50,7 @@ export function epochProgress(epochInfo: Network.EpochInfo) {
...
+        absoluteSlot,
         timeRemaining: {
           days: days,
           hours: hours,
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/WalletManager/network-manager/helpers/epoch-progress.ts]: ⬆️

File: packages/types/src/network/manager.ts (hunk 1 above)
Comment (stackchain) [reply packages/types/src/network/manager.ts:56]: hmmm eras looks fine

File: apps/wallet-mobile/src/features/WalletManager/network-manager/helpers/epoch-progress.ts (hunk 2 above)
Comment (michalszorad) [reply apps/wallet-mobile/src/features/WalletManager/network-manager/helpers/epoch-progress.ts]: Updated

File: apps/wallet-mobile/src/features/WalletManager/network-manager/helpers/epoch-progress.ts (hunk 3 above)
Comment (michalszorad) [reply apps/wallet-mobile/src/features/WalletManager/network-manager/helpers/epoch-progress.ts]: Updated

**Review by stackchain — APPROVED** [review]

**Pull request merged**